medical/models.py

```diff
@@ -164,12 +166,14 @@ class Service(VersionedModel, ItemOrService):
     code = models.CharField(db_column='ServCode', max_length=6)
     name = models.CharField(db_column='ServName', max_length=100)
     type = models.CharField(db_column='ServType', max_length=1)
+    packagetype = models.CharField(db_column='ServPackageType', max_length=1, default="S")
+    manualPrice = models.BooleanField(default=False)
     level = models.CharField(db_column='ServLevel', max_length=1)
     price = models.DecimalField(db_column='ServPrice', max_digits=18, decimal_places=2)
     maximum_amount = models.DecimalField(db_column='MaximumAmount', max_digits=18, decimal_places=2, blank=True, null=True)
     care_type = models.CharField(db_column='ServCareType', max_length=1)
     frequency = models.SmallIntegerField(db_column='ServFrequency', blank=True, null=True)
-    patient_category = models.SmallIntegerField(db_column='ServPatCat')
+    patient_category = models.SmallIntegerField(db_column='ServPatCat', default="15")
```

Comment:
Default should have a CONSTANT to make the code more readable

Reply:
Fixed